torchao/__init__.py

```diff
@@ -5,7 +5,10 @@
 )
 from . import dtypes
 import torch
-from . import _C
+from torch.testing._internal.common_utils import IS_FBCODE
+if not IS_FBCODE:
+    from . import _C
+
 from . import ops
```

Comment:
Might need to also bring this in under IS_FBCODE flag 

And also will need to skip this test https://github.com/pytorch/ao/blob/main/test/test_ops.py

Reply:
I'll probably need to test this in the fbcode-oss sync diff, since if I just import this PR it may not work since currently code base is not synced